### PR TITLE
Defined "Egonomics"

### LINKED ISSUE #241: Define egonomics
Submitting for consideration the following term and definition draft from the [DDH](https://medium.com/dictionary-of-digital-humanities):

**Egonomics**
The investment in time, effort, and money done by people to manage personal and professional online reputations.
_Term coined by [Thomas Schelling](https://en.wikipedia.org/wiki/Thomas_Schelling)_

### PR DESCRIPTION
**Pull request by eveline-xie**
**This PR address #241**  
 
**What it looks like on the index page**
index

**What it looks like on the definition page**
definition

### CODE REVIEW

File: 11ty/definitions/egonomics.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Egonomics
```
Comment (tatianamac) [inline 11ty/definitions/egonomics.md]:
```suggestion
title: egonomics
```

File: 11ty/definitions/egonomics.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+slug: egonomics
+defined: true
+excerpt: the investment in time, effort, and money done by people to manage personal and professional online reputations.
+speech: noun
+reading:
+  - text: Egonomics, or the Art of Self-Management
+    href: https://ideas.repec.org/a/aea/aecrev/v68y1978i2p290-94.html
+---
+
+The investment in time, effort, and money done by people to manage personal and professional online reputations.
```
Comment (tatianamac) [inline 11ty/definitions/egonomics.md]:
```suggestion
the investment in time, effort, and money done by people to manage personal and professional online reputations.
```

File: 11ty/definitions/egonomics.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+## Issues
```
Comment (tatianamac) [inline 11ty/definitions/egonomics.md]:
```suggestion
## Origins
```

**Review by tatianamac — APPROVED** [review]
Approved with a couple of small changes.

**Comment by tatianamac** [comment]
@eveline-xie Thank you for your immense patience with me and submitting! This definition is very close imho!

**Comment by tatianamac** [comment]
@eveline-xie Hi there! Do you still have interest/bandwidth to finish out this PR? Upon last read it was very close. Thank you!

**Comment by eveline-xie** [comment]
Sorry I'm not sure how I can improve the definition 😞

**Comment by tatianamac** [comment]
Ah! I'm so sorry @eveline-xie ! I think I misread the commit history. I'll merge this PR in. Thanks so much for your help and I hope you'll come back soon! 🙏🏽

**Pull request merged**